data/sysctl/rhel7-snap.conf

```diff
@@ -2,3 +2,10 @@
 # Unexpected "Device or resource busy" error when removing a directory
 # see https://access.redhat.com/articles/3128691 for details
 fs.may_detach_mounts=1
+
+# RHEL 7.4+ (and LXD) specific:
+# A configuration change was made in the 7.4 release, the default number of user
+# namespaces is set to zero, see https://access.redhat.com/solutions/3188102 for
+# details. For details about LXD side, see
+# https://discuss.linuxcontainers.org/t/lxd-on-centos-7/1250
+user.max_user_namespaces=15000
```

Comment:
It's a bit unfortunate we have to touch a global limit, but well... 
The forum topic you linked seems to be suggesting much lower value, any reason to go higher?

Reply:
That's the value suggested by RHEL knowledge base solution.